app/products/product-list.component.ts

```diff
@@ -13,6 +13,7 @@ import { ProductService } from './product.service';
 })
 export class ProductListComponent implements OnInit {
 
+  @HostBinding('class') classes = 'row';
```

Comment:
A way to add classes to your components.
Suggest how to add wrapper for list of products with class ".row"